app/js_service_encointer/src/service/account.js

```
@@ -227,69 +226,81 @@ export function sendTx (txInfo, paramList) {
-          if (txInfo.module === encointerBalances && txInfo.call === transfer) {
-            // make transfer amount human-readable again
-            paramList[2] = balanceHuman;
+      console.log(`[js-account/sendTx]: txInfo ${JSON.stringify(txInfo)}`);
+      console.log(`[js-account/sendTx]: Params ${JSON.stringify(paramList)}`);
+
+      const tx = api.tx[txInfo.module][txInfo.call](...paramList);
+      const onStatusChange = (result) => {
+        if (result.status.isInBlock || result.status.isFinalized) {
+          const {
+            success,
+            error
+          } = extractEvents(api, result);
+          if (success) {
+            if (txInfo.module === encointerBalances && txInfo.call === transfer) {
+              // make transfer amount human-readable again
+              paramList[2] = balanceHuman;
+            }
+
+            resolve({
+              hash: tx.hash.toString(),
+              time: new Date().getTime(),
+              params: paramList
+            });
           }
-
-          resolve({
-            hash: tx.hash.toString(),
-            time: new Date().getTime(),
-            params: paramList
-          });
-        }
-        if (error) {
-          resolve({ error });
+          if (error) {
+            resolve({ error });
+          }
+          unsub();
+        } else {
+          window.send('txStatusChange', result.status.type);
         }
-        unsub();
-      } else {
-        window.send('txStatusChange', result.status.type);
+      };
+      if (txInfo.isUnsigned) {
+        tx.send(onStatusChange)
+          .then((res) => {
+            unsub = res;
+          })
+          .catch((err) => {
+            resolve({ error: err.message });
+          });
+        return;
       }
-    };
-    if (txInfo.isUnsigned) {
-      tx.send(onStatusChange)
+
+      console.log(`[js-account/sendTx]: Adding tip: ${txInfo.tip}`);
+      const signerOptions = {
+        tip: new BN(txInfo.tip || 0, 10)
```

This was the second error that I had after the update. `BN` does no longer default to 0 with null, and it throws a very misleading `invalid character error`.